<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mer</class>
 <widget class="QDialog" name="Mer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>586</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="edtFileName">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>30</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>33</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SEG-Y File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpen">
   <property name="geometry">
    <rect>
     <x>404</x>
     <y>28</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCaculate">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>128</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QwtPlot" name="PlotMer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>761</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>483</x>
     <y>30</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Trace NO.</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinTraceNo">
   <property name="geometry">
    <rect>
     <x>537</x>
     <y>27</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000000</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>171</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Algorithm</string>
   </property>
   <widget class="QRadioButton" name="radioER">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Energy Ratio</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioMER">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Modified Energy Ratio</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioStaLta">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>STA/LTA</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>14</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Display Mode</string>
   </property>
   <widget class="QRadioButton" name="radioOverlap">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Overlap</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioIndividual">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Individual</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>157</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>130</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>STA/LTA</string>
   </property>
   <widget class="QLineEdit" name="edtShortWindow">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>19</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>203</x>
      <y>21</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Short Window</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtLongWindow">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>20</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Long   Window</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>70</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>ER/MER</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Window Length</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtWindowLen">
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>18</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lbSampleRate">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>560</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample Rate:</string>
   </property>
  </widget>
  <zorder>groupBox_3</zorder>
  <zorder>edtFileName</zorder>
  <zorder>label</zorder>
  <zorder>btnOpen</zorder>
  <zorder>btnCaculate</zorder>
  <zorder>PlotMer</zorder>
  <zorder>label_3</zorder>
  <zorder>spinTraceNo</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>btnClear</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>lbSampleRate</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
